content/courses/intro-to-ada/chapters/modular_programming.rst

```diff
@@ -199,7 +199,7 @@ user of the :ada:`Operations` package, providing a first form of encapsulation.
 This works because entities declared in the body are *only* visible in the
 body.
 
-This example shows how :ada:`Last_Increment` is used indirectly:
+This example shows how :ada:`Increment_By` is used indirectly:
```

Comment:
`Last_Increment` was actually correct, as we're referring to the variable, not the function.